# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/common/angleutils.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/common/angleutils.h
--- a/gfx/angle/checkout/src/common/angleutils.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/common/angleutils.h	Sat Sep 07 13:38:36 2019 +0000
@@ -324,4 +324,10 @@
 #    define ANGLE_NO_DISCARD
 #endif  // __has_cpp_attribute(nodiscard)
 
+#if __has_cpp_attribute(maybe_unused)
+#    define ANGLE_MAYBE_UNUSED [[maybe_unused]]
+#else
+#    define ANGLE_MAYBE_UNUSED
+#endif  // __has_cpp_attribute(maybe_unused)
+
 #endif  // COMMON_ANGLEUTILS_H_